Doc/c-api/init.rst

```diff
@@ -373,6 +373,14 @@ Initializing and finalizing the interpreter
    :c:func:`Py_Initialize` is called again.
 
 
+.. c:function:: int Py_IsFinalizing()
```

Comment:
What is the order of functions here, lexicographical or functional? Should not it be below `Py_FinalizeEx()`?

Reply:
For me, the order is more logical:

* Functions to initialize Python (Py_Initialize)
* Functions to check if Python is being shutting down (Py_IsFinalizing)
* Functions to shutdown Python (Py_Finalize)

Reply:
Any order is good if it is deliberate.